overlord/snapstate/snapmgr.go

```diff
@@ -1029,6 +1140,23 @@ func (m *SnapManager) startSnapServices(t *state.Task, _ *tomb.Tomb) error {
 	return err
 }
 
+func scheduleNextRefresh(st *state.State) time.Time {
+	randomness := rand.Int63n(int64(refreshRandomness))
+	nextRefreshTime := time.Now().Add(refreshInterval).Add(time.Duration(randomness))
```

Comment:
this behaves a bit differently than RandomizedDelaySec, wondering if we should use a smaller a randomess